docs/src/content/docs/challenges/angular/45-react-in-angular.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 🔴 React in angular
+description: Challenge 45 is about learning how to benefit from the numerous libraries in React
+author: Wandrille Guesdon
```

Comment:
the author must match the name of the file

Reply:
Ah, well noticed! thanks. It's fixed